src/pages/Teachers/Teachers.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { useEffect } from "react";
+import { connect } from "react-redux";
+import fetchTeachers from "../../services/teachersRequest";
+
+import Teacher from "../../components/Teacher";
+
+import "./teachers.scss";
+
+const Teachers = ({ teacherData, fetchTeachers }) => {
+  useEffect(() => {
+    fetchTeachers();
+  }, [fetchTeachers]);
+  return (
+    <section className="teachers">
+      <div className="container">
+        <div className="teachers__body">
+          <h1 className="teachers__body-title">Meet our great team!</h1>
+          <div className="teachers__body-main">
+            {teacherData.teachers.map((teach) => (
```

Comment:
and avoind underscore naming
teachers.map((teacher) => <Component {...teacher} />)